subsys/bluetooth/audio/Kconfig.bap

```diff
@@ -166,7 +166,7 @@ config BT_BAP_BROADCAST_SINK
 	bool "Bluetooth Broadcast Sink Audio Support"
 	select BT_ISO_SYNC_RECEIVER
 	select BT_AUDIO_RX
-	select BT_PAC_SNK
+	depends on BT_PAC_SNK
```

Comment:
I assume this is due some circular dependency?

Reply:
Yes, that's true.